bootstrap/index.ts

```diff
@@ -27,13 +28,22 @@
             }
         }
 
-        const aliucordDir = externalStorageDirectory + "/AliucordRN";
+        const aliucordDir = `${externalStorageDirectory}/AliucordRN`;
         AliuFS.mkdir(aliucordDir);
 
-        const bundlePath = aliucordDir + "/Aliucord.js.bundle";
-        if (!AliuFS.exists(bundlePath)) await download("https://raw.githubusercontent.com/Aliucord/AliucordRN/builds/Aliucord.js.bundle", bundlePath);
+        const commits = await fetch("https://api.github.com/repos/Aliucord/AliucordRN/commits?sha=builds");
```

Comment:
use the commit hash for the file itself + max limit 1

like this

https://github.com/Aliucord/AliucordManager/blob/4bb45b4ea711eec499d4da7424c96263ed930ca3/app/src/main/kotlin/com/aliucord/manager/network/service/AliucordGithubService.kt#L25-L27

Reply:
this way it doesn't update every single time something else in builds updates